easybuild/easyblocks/p/python.py

```diff
@@ -250,3 +254,18 @@ def sanity_check_step(self):
                 raise EasyBuildError("Expected to find exactly one _tkinter*.so: %s", tkinter_so_hits)
 
         super(EB_Python, self).sanity_check_step(custom_paths=custom_paths, custom_commands=custom_commands)
+
+    def make_module_req_guess(self):
+        """
+        Return dictionary with additional entries for path-like environment variables;
+        also include <prefix>/include/python* to $CPATH.
+        """
+        guesses = super(EB_Python, self).make_module_req_guess()
+
+        pyincdir = os.path.join('include', 'python' + self.pyshortver)
+        if LooseVersion(self.version) >= LooseVersion('3'):
+            pyincdir += 'm'
```

Comment:
In the issue thread, mstud said that you need to "add the abiflags to the version,". I guess that's the "m", but is that the only possibility or is there something else that could be the right thing here?

Reply:
That's a good question...

I guess a safer approach could be to use a glob pattern to figure out the name of the `include/python<version>*` directory. Hwo does that sound?

Reply:
Good enough I guess.